src/dune/dune_file.ml

```diff
@@ -424,6 +448,8 @@ module Buildable = struct
       field "js_of_ocaml" Js_of_ocaml.decode ~default:Js_of_ocaml.default
     and+ allow_overlapping_dependencies =
       field_b "allow_overlapping_dependencies"
+    and+ bisect_ppx =
+      field_b "bisect_ppx" ~check:(Dune_lang.Syntax.since Stanza.syntax (2, 5))
```

Comment:
This should be `(2, 6)` since 2.5 is the last released version. You'll need to bump the `latest_version` variable in `stanza.ml`.

Additionally, I just remembered that we need to guard this field by a `bisect_ppx` extension. i.e. the `bisect_ppx` field in `dune` files should only be available if the user wrote `(using bisect_ppx 1.0)` in their `dune-project` file. This gives more flexibility and allows to desynchronise the syntax of the `bisect_ppx` field from the main vanilla language. This i important because this functionality relies on an external tool that has its own release cycle.

To do that, add the following definition at the toplevel:

```ocaml
let bisect_ppx_syntax =
  Dune_lang.Syntax.create ~name:"bisect_ppx" ~desc:"the bisect_ppx extension"
    [ ((1, 0), `Since (2, 6)) ]

let () =
  Dune_project.Extension.register_simple bisect_ppx_syntax
    (Dune_lang.Decoder.return [])
```

and then replace the above `Stanza.syntax (2, 5)` by `bisect_ppx_syntax (1, 0)`.